docker-compose.yml

```diff
@@ -118,6 +118,15 @@ services:
       # - TTN_LW_AS_MQTT_PUBLIC_ADDRESS=thethings.example.com:1883
       # - TTN_LW_AS_MQTT_PUBLIC_TLS_ADDRESS=thethings.example.com:8883
 
+      # If Application Server enabled, Application Server interoperability configuration:
```

Comment:
The docker-compose example always has the AS enabled, so I think it's better if we leave this out to avoid confusion.

Reply:
I agree about consistency; this change is good.

If we remove the `If <component> enabled,` pieces, we can do that separately